README.md

```diff
@@ -125,3 +125,20 @@ Configuring Shibboleth
 * visit '/Shibboleth.sso/Metadata' to download and review the metadata, rename to a filename of your preference
 * upload the SP metadata to http://testshib.org/register.html or your Identiy Provider (IdP)
 * login by clicking on 'Sign into Shibboleth' and choosing one of the available identities
+
+Audit Fix
+---
+* To fix data with triple files:
+```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'triples','sample-sparql.txt'```
+* To remove irrelevant theses dates:
+```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'thesis'```
```

Comment:
try the attached foxml for the testing. This fix is to remove the theses
related date fields from non-thesis objects, so we need a non-thesis object
to test:


*Weiwei Shi*

Digital Initiative Applications Librarian
University of Alberta Libraries
2-10L Cameron Library
Edmonton, Alberta, Canada T6G 2J8
Phone:(780)492-7802
Fax: (780)248-1209
Email: weiwei.shi@ualberta.ca

On Wed, Jun 1, 2016 at 3:20 PM, pgwillia <notifications@github.com> wrote:

> In README.md
> <https://github.com/ualbertalib/HydraNorth/pull/1083#discussion_r65444943>
> :
>
> > @@ -125,3 +125,20 @@ Configuring Shibboleth
> >  * visit '/Shibboleth.sso/Metadata' to download and review the metadata, rename to a filename of your preference
> >  * upload the SP metadata to http://testshib.org/register.html or your Identiy Provider (IdP)
> >  * login by clicking on 'Sign into Shibboleth' and choosing one of the available identities
> > +
> > +Audit Fix
> > +---
> > +* To fix data with triple files:
> > +```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'triples','sample-sparql.txt'```
> > +* To remove irrelevant theses dates:
> > +```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'thesis'```
>
> rake
> migration:eraitem['spec/fixtures/migration/test-metadata/legacy-thesis-metadata/']
> gave me a thesis to test with. Unfortunately I get the same error as before.
>
> [root@hydranorthdev hydranorth]# bundle exec ruby bin/fix/fix-audit.rb 'thesis'
> bin/fix/fix-audit.rb:73:in `find_objects': undefined method `length' for nil:NilClass (NoMethodError)
>     from bin/fix/fix-audit.rb:111:in `fix_theses_related_dates'
>     from bin/fix/fix-audit.rb:180:in `<main>'
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/pull/1083/files/e06e6ae74fbeeedf40f19e4e2febbbf29a19e63a#r65444943>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB8-fuNdT5KcKKSIsBECRR00iFbYfPHwks5qHfehgaJpZM4IZLGv>
> .
>